tethys_gizmos/static/tethys_gizmos/less/tethys_map_view.less

```diff
@@ -75,7 +75,6 @@
 
   .tethys-map-view-draw-icon {
     display: inline-block;
-    background: url('/static/tethys_gizmos/images/map-view-drawing-icons.gif') no-repeat;
```

Comment:
I'm not sure what this style is used for, but does removing the background image make sense?

Reply:
i.e. do we need to find another way to get the URL for the background image?

Reply:
Ah, I found it. You've just put this directly in the template. Maybe a comment here would be a helpful clarification?

Reply:
We had to move it to the template so that we could retrieve it dynamically, rather than having it hard-coded.

Reply:
> Ah, I found it. You've just put this directly in the template. Maybe a comment here would be a helpful clarification?

@romer8 could you add a comment where we removed this line that references the template it is in?

Reply:
Perfect, I have added some comments in the palces where the code was removed and added.